spec/index.bs

```diff
@@ -2509,6 +2520,12 @@ enum GPUPowerPreference {
         [=fallback adapter=]. Developers that wish to prevent their applications from running on
         [=fallback adapters=] should check the {{GPUAdapter}}.{{GPUAdapter/isFallbackAdapter}}
         attribute prior to requesting a {{GPUDevice}}.
+
+    : <dfn>xrCompatible</dfn>
+    ::
+        When set to `true` indicates that the best [=adapter=] for rendering to a [=WebXR session=]
+        must be returned. If the user agent or system does not support [=WebXR sessions=] then
+        adapter selection may ignore this value.
```

Comment:
If you don't request `xrCompatible`, does that prevent you from using it with WebXR?

Regardless of the answer, I think it's worth a spec mention here.

Reply:
Added a note.